specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.6/TextAnalytics.json

```diff
@@ -0,0 +1,3282 @@
+            "after-start",
+            "before-approx",
+            "mid",
+            "more"
+          ],
+          "x-ms-enum": {
+            "name": "modifier",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "timex",
+        "subType",
+        "value"
+      ]
+    },
+    "DimensionFields": {
+      "type": "object",
+      "properties": {
+        "subType": {
+          "type": "string",
+          "enum": [
+            "Speed",
+            "Weight",
+            "Volume",
+            "Length",
+            "Area",
+            "Information"
+          ],
+          "x-ms-enum": {
+            "name": "subType",
+            "modelAsString": true
+          },
+          "description": "represents the Dimension subType (e.g., Speed, Volume, Weight)"
+        },
+        "unit": {
+          "type": "string",
+          "description": "The measured unit"
+        },
+        "value": {
+          "type": "string",
+          "description": "The actual value that the extracted text denotes."
+        }
+      },
+      "required": [
+        "subType",
+        "unit",
+        "value"
+      ]
+    },
+    "TemperatureFields":{
+      "type": "object",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "F",
+            "C",
+            "Degree"
+          ],
+          "x-ms-enum": {
+            "name": "subType",
+            "modelAsString": true
+          },
+          "description": "The temperature unit"
+        },
+        "value": {
+          "type": "string",
+          "description": "The actual temperature value that the extracted text represents."
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "CurrencyFields":{
+      "type": "object",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "description": "The Currency unit"
+        },
+        "value": {
+          "type": "string",
+          "description": "The actual currency value that the extracted text denotes."
+        }
+      },
+      "required": [
+        "value",
+        "unit"
+      ]
+    },
+    "AgeFields":{
+      "type": "object",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Year",
+            "Month",
+            "Week",
+            "Day",
+            "Age"
+          ],
+          "x-ms-enum": {
+            "name": "unit",
+            "modelAsString": true
+          },
+          "description": "The age unit"
+        },
+        "value": {
+          "type": "string",
+          "description": "The actual age value that the extracted text denotes."
+        }
+      },
+      "required": [
+        "value",
+        "unit"
+      ]
+    },
+    "NumberFields":{
+      "type": "object",
+      "properties": {
+        "numType": {
+          "type": "string",
+          "enum": [
+            "integer",
```

Comment:
Is there a reason why you have these enum values as lowercase while other enums are uppercase?

Reply:
What is the convention? first letter uppercase, the rest small or camelCase?